src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,83 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private Entry<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        this.table = new Entry[DEFAULT_CAPACITY];
```

Comment:
```suggestion
        table = new Entry[DEFAULT_CAPACITY];
```
`this` is not needed here, here's [where ](https://stackoverflow.com/a/2411283) we need it